vfs/fs_utils.h

```diff
@@ -48,6 +48,8 @@ struct FsObjectInfo {
     std::map<std::string, Json::Value> objectMetadata;  ///< Metadata from obj
 
     operator bool () const { return exists; }
+
+    std::string what;                ///< MLDB-2150, hack, do not use
```

Comment:
Instead of that, why don't we simply call it errorMessage.  Then it's more generic and we can use it.  What is a terrible name.